bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/CoolBar.java

```diff
@@ -1198,4 +1199,47 @@ LRESULT wmNotifyChild (NMHDR hdr, long wParam, long lParam) {
 	}
 	return super.wmNotifyChild (hdr, wParam, lParam);
 }
+
+private static void handleDPIChange(Widget widget, int newZoom, float scalingFactor) {
```

Comment:
I don't think the usage of `var` on the variable of this method has any advantages. In fact it only makes it a bit more difficult to know the type of the variables.
IMO `var` should be used only where the type doesn't really play a role but these variables are either being passed to other methods or used inside calculations inside this method, so it's useful to know at first glance of which type the are. Additionally, it introduces another risk: imagine someone changes the return type of a method that lands in one of these `var`-variables (say `CoolBar::getWrapIndices`) and there is an overloaded method which happens to work with that new type. If that happens then you will inadvertently end up calling another method.